guides/node/cosmovisor.md

```diff
@@ -0,0 +1,76 @@
+
+## Install Cosmovisor
+
+[Cosmovisor](https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor) is a process manager for running application binaries. Using Cosmovisor is not required but recommended for node operators that would like to automate the upgrade process.
+
+To install `cosmovisor`, run the following command:
+
+```bash
+go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v1.0
+```
+
+Check to ensure the installation was successful:
+
+```bash
+cosmovisor version
```

Comment:
This command will return an error if the env variables are not set. Even after setting the env variables it will return the version of `akash` being used. To verify the version of `cosmovisor` being used we can execute this command:- `strings $(which cosmovisor) | egrep -e "mod\s+github.com/cosmos/cosmos-sdk/cosmovisor"`

It should return a response like this 
`mod	github.com/cosmos/cosmos-sdk/cosmovisor	v1.0.0	h1:80NRDDUnfuVB9+PJD5Z2sK1b6A4r5r5H1SqgDR5ejmo=`

Reply:
In cosmovisor 1.0.0, this is fixed. cosmovisor should give it's own version now. But yeah, we need to export the variables before hand